core/sql/optimizer/OptimizerSimulator.cpp

```diff
@@ -278,9 +277,10 @@ OptimizerSimulator::OptimizerSimulator(CollHeap *heap)
     sysCallLogFilePath_[sc]=NULL;
     writeSysCallStream_[sc]=NULL;
   }
+  // Init log4cxx logging
+  CommonLogger::instance().initLog4cxx("log4cxx.trafodion.osim.config");
```

Comment:
Yes, OSIM is running in compiler.
The initialization process is not necessary, but we do need a separate logger to print OSIM message to console.